src/components/plot/plot.scss

```diff
@@ -37,12 +37,14 @@
   float: right;
 }
 
-.bookmark-button, .copy-button {
+.command {
   cursor: pointer;
-  margin-left: 4px;
+  margin-left: 20px;
```

Comment:
I found myself accidentally click on the icon next to it, so I made the margin larger. 

Screenshot:
![image](https://user-images.githubusercontent.com/19985016/28740508-69f8aec8-73ba-11e7-9d91-3cbcee919fc9.png)